src/main/types/OpeningTime.ts

```diff
@@ -0,0 +1,19 @@
+import {SelectItem} from "./CourtPageData";
+
+export interface OpeningTime {
+  type_id: number,
+  hours: string
+}
+
+export interface OpeningType {
```

Comment:
Is it worth putting this into its own file? I.e OpeningType.ts ? Only a small one, not to worry if its a pain

Reply:
Agreed - code updated with separate file for this type.